quickwit/quickwit-control-plane/src/scheduler.rs

```diff
@@ -42,10 +42,21 @@ use crate::indexing_plan::{
 };
 use crate::{NotifyIndexChangeRequest, NotifyIndexChangeResponse};
 
+/// Interval between two controls (or checks) of the desired plan VS running plan.
+const CONTROL_PLAN_LOOP_INTERVAL: Duration = if cfg!(any(test, feature = "testsuite")) {
+    Duration::from_millis(500)
+} else {
+    Duration::from_secs(3)
+};
+
+/// Interval between two scheduling of indexing plans. No need to be faster than the
+/// control plan loop.
+// Note: it's currently not possible to define a const duration with
+// `CONTROL_PLAN_LOOP_INTERVAL * number`.
 const REFRESH_PLAN_LOOP_INTERVAL: Duration = if cfg!(any(test, feature = "testsuite")) {
     Duration::from_secs(3)
 } else {
-    Duration::from_secs(60 * 5)
+    Duration::from_secs(60)
```

Comment:
Ok... it's just me here... I prefer to check this every minute... one refresh is making one call. I can revert if needed.